### PR TITLE
 Fixed toplevel msg with RAKE_DIR

### PR DESCRIPTION
**Pull request by jasnow**
Fixed toplevel msg with RAKE_DIR as recommended. Removed my customization.

Tested another repo using
 * gem "sorbet-rails", path: '../sorbet-rails'

and ran:
 * rake rails_rbi:all

and got:
```
Note: Unable to create new instance of ActiveStorage::Attachment
-- Generate sigs for ActiveStorage::Attachment --
Note: Unable to create new instance of ActiveStorage::Blob
-- Generate sigs for ActiveStorage::Blob --
-- Generate sigs for Micropost --
-- Generate sigs for User --
-- Generate sigs for ActiveRecord::InternalMetadata --
-- Generate sigs for ActiveRecord::SchemaMigration --
-- Generate sigs for helpers --
```
and "git status" got:
```
	modified:   Gemfile
	modified:   Gemfile.lock
	modified:   sorbet/rails-rbi/models/active_record/internal_metadata.rbi
	modified:   sorbet/rails-rbi/models/active_record/schema_migration.rbi
	modified:   sorbet/rails-rbi/models/active_storage/attachment.rbi
	modified:   sorbet/rails-rbi/models/active_storage/blob.rbi
	modified:   sorbet/rails-rbi/models/micropost.rbi
	modified:   sorbet/rails-rbi/models/user.rbi

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	sorbet/rails-rbi/helpers.rbi
```

### CODE REVIEW

File: lib/sorbet-rails/tasks/rails_rbi.rake [hunk 1]
```diff
@@ -5,7 +5,7 @@ require("sorbet-rails/utils")
 
 # this is ugly but it's a way to get the current directory of this script
 # maybe someone coming along will know a better way
-@@sorbet_rails_rake_dir = File.dirname(__FILE__)
+RAKE_DIR  = File.dirname(__FILE__)
```
Comment (manhhung741) [inline lib/sorbet-rails/tasks/rails_rbi.rake]:
```suggestion
RAILS_RBI_RAKE_DIR  = File.dirname(__FILE__)
```

File: lib/sorbet-rails/tasks/rails_rbi.rake [hunk 2]
```diff
@@ -67,7 +67,7 @@ namespace :rails_rbi do
   end
 
   def copy_bundled_rbi
-    bundled_rbi_path = File.join(@@sorbet_rails_rake_dir, "..", "rbi", ".")
+    bundled_rbi_path = File.join(RAKE_DIR, "..", "rbi", ".")
```
Comment (manhhung741) [inline lib/sorbet-rails/tasks/rails_rbi.rake]:
```suggestion
    bundled_rbi_path = File.join(RAILS_RBI_RAKE_DIR, "..", "rbi", ".")
```

**Comment by manhhung741** [comment]
Could you please accept my suggestion on variable name change? Once it's done I'll merge the PR

File: Gemfile [hunk 3]
```diff
@@ -48,4 +48,6 @@ else
   # prefer to test against latest version because sorbet is updated frequently
   gem 'sorbet'
   gem 'sorbet-runtime'
-end
\ No newline at end of file
+  gem 'dalli' # Needed by sorbet.
```
Comment (manhhung741) [inline Gemfile]: Does these lines resolve any warning? If yes, maybe we should keep them. I've meant to clean up the warning/unnecessary messages but I haven't done it yet.

File: lib/sorbet-rails/tasks/rails_rbi.rake (hunk 1 above)
Comment (jasnow) [reply lib/sorbet-rails/tasks/rails_rbi.rake]: Will change name later tonight

File: lib/sorbet-rails/tasks/rails_rbi.rake (hunk 2 above)
Comment (jasnow) [reply lib/sorbet-rails/tasks/rails_rbi.rake]: Will change name later tonight.

File: Gemfile (hunk 3 above)
Comment (jasnow) [reply Gemfile]: Let's work them separately.

**Comment by jasnow** [comment]
done

**Comment by manhhung741** [comment]
This will fix #90

**Comment by manhhung741** [comment]
Thank you very much @jasnow

**Comment by jasnow** [comment]
You are welcome

**Comment by jasnow** [comment]
The real thanks goes to @palasha for giving me the solution.
@palasha - THANKS for helping out!

**Pull request merged**